src/components/Notification/subcomponents/CompactNotification/CompactNotification.tsx

```diff
@@ -0,0 +1,76 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import * as React from 'react';
+import {
+  actionsContainer,
+  infoContainer,
+  notificationsContainer,
+  iconContainer,
+  primaryActionContainer,
+  closeActionContainer,
+} from './CompactNotification.style';
+import Icon from '../../../Icon';
+import { AcceptedColorComponentTypes } from 'utils/themeFunctions';
+import { NotificationTypes } from '../../Notification';
+
+export type CompactNotificationVariants = 'inline';
+
+export type Props = {
+  /** The informative message of the Notification */
+  message: string;
+  /** The variant of the Notification */
+  variant: CompactNotificationVariants;
+  /** The type of the Notification */
+  type: NotificationTypes;
+  /** Show notification icon based on the type */
+  withIcon: boolean;
+  /** Use color filling */
+  withFilling: boolean;
+  /** The primary call-to-action label of the Notification */
+  primaryCTALabel?: string;
+  /** The primary call-to-action of the Notification */
+  primaryCTA?: () => void;
+  /** The closing call-to-action of the Notification */
+  closeCTA?: () => void;
+};
+
+const typeToColor = (type: string): AcceptedColorComponentTypes =>
```

Comment:
with the change i suggested at `CompactNotificationVariants` to change alert with warning that is the same thing with different naming this is not needed anymore

Reply:
Icon Component's props "name" and "color" accept the same value for all NotificationTypes except "warning":
Icon:name expects "alert" while Icon:color expects "warning"
Therefore, this converter is needed (updated to colorToIconName)

Reply:
if you change `NotificationTypes` as i suggested you will see what i mean. The problem here is that you use different types and specifically the text `alert`. Alert in our system is called warning.